frontend/src/components/navbar/nav.module.css

```diff
@@ -0,0 +1,75 @@
+.navLinkContainer {
+  padding: 1.5rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  background-color: #110d59;
+  margin-top: 0;
+}
+.header {
+  font-size: 1.2rem;
+  display: flex;
+  align-items: center;
+  margin-left: 1rem;
+  color: #f5f5f5;
+  height: 64px;
+}
+
+.header h1 {
+  font-size: 1.5rem;
+  font-weight: bold;
+  margin-left: 1rem;
+}
+
+.icon {
+  width: 1.5rem;
+  height: 1.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.icon img {
+  width: 1.2rem;
+  height: 1.2rem;
+  margin-right: 0.5rem;
+}
+
+.navLink {
+  padding: 0.5rem 1rem;
+  background-color: #110d59;
+  border-radius: 0.4rem;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  transition:
+    background-color 0.3s ease,
+    color 0.3s ease;
+  color: #f5f5f5;
+}
+
+.navLink .icon {
+  margin-right: 1rem;
+}
+
+.navLink:hover {
+  background-color: #f5f5f5;
+  color: #110d59;
+  transition:
+    background-color 0.3s ease,
+    color 0.3s ease;
+}
+.link {
+  text-decoration: none;
+}
+.selectedLink {
+  background-color: #f5f5f5;
+  color: #110d59;
+}
+@media screen and (max-width: 930px) {
+  .navLink {
+    font-size: 14px;
+    margin-bottom: 1rem;
```

Comment:
when you delete this, it looks better I think
![Screenshot 2023-10-02 at 17 16 35](https://github.com/HackYourFuture-CPH/class-25-ironside/assets/48878547/d4762286-5b6e-4c4a-abbc-eea607156d24) what do you think? @hamoudi-hamoudi

Reply:
ok , I will try it that way

Reply:
and actually it looks better without the spacing from figma